_helpers/meta-template/README.md

```diff
@@ -6,7 +6,7 @@
 
 ### Environment variables
 
-This project requires some environment variables to be set. To keep your tokens and secrets secure, make sure to not commit the `.env` file in git. When setting up the project with `twilio serverless:init ...` the Twilio CLI will create a `.gitignore` file that excludes `.env` from the version history.
+This project requires some environment variables to be set. A file named `.env` is used to store the values for those environment variables, and you can copy or rename the `.env.example` file to `.env` to set up skeleton and default values for each variable. To keep your tokens and secrets secure, make sure to not commit the `.env` file in git. When setting up the project with `twilio serverless:init ...` the Twilio CLI will create a `.gitignore` file that excludes `.env` from the version history.
```

Comment:
The serverless toolkit will actually create an `.env` file regardless of whether the template has a `.env` file or a `.env.example` file. [See here for the code](https://github.com/twilio-labs/serverless-toolkit/blob/main/packages/twilio-run/src/templating/filesystem.ts#L157-L165).

Reply:
I added a little bit of text describing how (I think) deployed Functions pick up values from `.env` vs `.env.example`.

Reply:
I think you've gone in the wrong direction with that update.

When you load a function template using `twilio serverless:init --template` or `twilio serverless:new` then regardless of whether the template has an `.env` or `.env.example` file, the project will only result in a `.env` file. `twilio serverless` commands do not create a `.env.example` file.